sql/log.cc

```diff
@@ -3571,16 +3571,6 @@ void MYSQL_BIN_LOG::cleanup()
 }
 
 
-/* Init binlog-specific vars */
-void MYSQL_BIN_LOG::init(ulong max_size_arg)
```

Comment:
Ok!

Reply:
The API argument should not apply here. E.g mysqlbinlog is an external program and would obviously feel good.